docs/08-02-cli-in-serverless.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Use Kyma CLI to manage Functions
+type: Tutorials
+---
+
+This tutorial shows how to use the available CLI commands to manage serverless Functions in Kyma. You will see how to:
+
+1. Create local files that contain the basic configuration for a sample "Hello World" Python Function (`kyma init function`).
+2. Generate a Function custom resource from these files and apply it on your cluster (`kyma apply function`).
+3. Fetch the current state of your Function's cluster configuration after it was modified (`kyma sync function`).
+
+This tutorial is based on a sample Python Function run on a lightweight [k3d](https://k3d.io/) cluster.
+
+## Prerequisites
+
+Before you start, make sure you have the following tools installed:
+
+- [Docker](https://www.docker.com/)
+- [k3d](https://k3d.io/)
+- Kyma CLI
+
+## Steps
+
+1. Create a default k3d cluster with a single server node:
+
+  ```bash
+  k3d cluster create {CLUSTER_NAME}
+  ```
+
+2. Apply the `functions.serverless.kyma-project.io` CRD from sources in the [`kyma`](https://github.com/kyma-project/kyma/tree/master/resources/cluster-essentials/files) repository. You will need it to create a Function CR on the cluster based on this definition.
+
+  ```bash
+  kubectl apply -f https://raw.githubusercontent.com/kyma-project/kyma/master/resources/cluster-essentials/files/functions.serverless.crd.yaml
+  ```
+
+3. Use the `init` Kyma CLI command to create local files with the default configuration for a Python Function. Go to the folder in which you want to initiate the workspace content and run this command:
+
+  ```bash
+  kyma init function --runtime python38 --name {FUNCTION_NAME}
+  ```
+
+  Alternatively, use the `--dir {FULL_FOLDER_PATH}` flag to point to the directory where you want to save the fetched Function's source files.
+
+  > **NOTE:** Python 3.8 is only one of the available runtimes. Read about all [supported runtimes and sample Functions to run on them](https://kyma-project.io/docs/master/components/serverless/#details-runtimes).
+
+  The `init` command downloads the following files to your workspace folder:
+
+    - `config.yaml`	with the Function's configuration
```

Comment:
FYI, this whole list is displayed as a code block in GitHub Preview. However, on Netlify preview it looks good so maybe there's nothing to worry about 🙂

Reply:
I'll try to fix that, thx :)